app/lib/common-utils.js

```diff
@@ -1,8 +1,33 @@
+const usersApi = require('../core/api').users
+const localStorage = require('../core/storage')
+
 function validateEmail(email) {
   const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
   return re.test(String(email).toLowerCase());
 }
 
+function provisionPremium () {
```

Comment:
I feel like these functions should either go in a library-specific lib, the User model, or a user-specific lib. If we keep them in `lib/common-utils.js`, how do we differentiate from `core/utils.coffee`, other than we kinda want to write new stuff in JS? What should our organize scheme be for this proliferation of various libs?